docs/source/sections/Command_line_interface.rst

```diff
@@ -21,6 +21,7 @@ positional arguments:
     config : A configuration file detailing how to build the digital environment. JSON parsable
```

Comment:
The intro to this section needs to be updated. "CaartograPhi" should be MeshiPhi and we should describe meshing without reference routing.